apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js

```diff
@@ -34,8 +37,7 @@ describe('The JSDebugger component', () => {
     getStore().dispatch(actions.initialize({runApp}));
     getStore().dispatch(actions.open());
 
-    addEventSpy = sinon.spy(document.body, 'addEventListener');
-    removeEventSpy = sinon.spy(document.body, 'removeEventListener');
+    ({addEventSpy, removeEventSpy} = getBodyEventSpies());
```

Comment:
it seems strange to me that the outer parens are necessary, but the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) seem to agree.

Reply:
Yeah, this syntax is a little gross and uncommon in our code.  I considered removing the `getBodyEventSpies()` helper and calling the inner helper directly (twice) in the test setup code, removing this weird destructuring.  Do you think that would be better?

Reply:
No, I think the destructuring assignment is fine -- it's clear what is happening, and seems better than any alternative for returning two values in a single function call. I was just surprised the seemingly unnecessary outer parens were required by the language. It would also be fine to eliminate it by making two separate function calls as you described, though.